WalletWasabi.Documentation/Ports.md

```diff
@@ -17,6 +17,8 @@ HiddenWallet's ports (3712x) are chosen within a long range of unassigned IANA p
 |37129 | Wallet Wasabi Local Client TCPListener on MainNet |
 |37130 | Wallet Wasabi Local Client TCPListener on TestNet |
 |37131 | Wallet Wasabi Local Client TCPListener on RegTest |
+|37150 | Tor socks port used by Wallet Wasabi |
+|37151 | Tor control port used by Wallet Wasabi |
 |9050  | Default Tor socks port |
```

Comment:
Should this get removed?

Reply:
?

Reply:
Yahia says that `|9050  | Default Tor socks port |` is no longer in use IMO.

Reply:
I see. No this shouldn't be removed.